[Learn][PWA] Replacing WebM with iOS-friendly MP4 (#9108)
GoogleChrome/web.dev: src/site/content/en/learn/pwa/windows/index.md

> An experimental capability, known as *tabbed mode*​ will let your PWA have a tab-based design similar to a web browser. In this case, the user can have several tabs opened from the same PWA but all tied together in the same operating system window, as you can see in the following image:

Reviewer: Might wanna update this, since it's no longer just an image:

```suggestion
An experimental capability, known as *tabbed mode*​ will let your PWA have a tab-based design similar to a web browser. In this case, the user can have several tabs opened from the same PWA but all tied together in the same operating system window, as you can see below:
```

{% Video src="video/W3z1f5ZkBJSgL1V1IfloTIctbIF3/I0idt5S5IvAu5iy0WCq7.mp4",
autoplay="true",
loop="true",

> muted="true",

Reviewer: Remove trailing comma, if removing `alt`, `height`, and `width` as suggested.

```suggestion
muted="true"
```

> muted="true"
> %}

Reviewer: Tested this and it STILL doesn't work :-(

Apparently [iOS won't allow autoplaying](https://developer.apple.com/documentation/webkit/delivering_video_content_for_safari#3030251) unless you also specify `playsinline`.

Probably want to add controls to allow it to stop playing while you're at it:

```suggestion
muted="true",
playsinline="true",
controls="true"
%}
```

> autoplay="true",
> loop="true",
> muted="true"
> %}

Reviewer: Tested this and it STILL doesn't work :-(

Apparently [iOS won't allow autoplaying](https://developer.apple.com/documentation/webkit/delivering_video_content_for_safari#3030251) unless you also specify `playsinline`.

Probably want to add controls to allow it to stop playing while you're at it. But annoyingly that shows the controls by default which is a little ugly. So next best option I've got is to not autoplay but instead add a poster image:

```suggestion
loop="true",
muted="true",
playsinline="true",
controls="true",
poster="image/W3z1f5ZkBJSgL1V1IfloTIctbIF3/emg7quu4DyvYwlyzgDE1.png"
%}
```

> alt="Video showing the experimental tab-based design", width="800", height="519" %}

Reviewer: You need to remove height and width for now, or you get a massive gap:

```suggestion
alt="Video showing the experimental tab-based design" %}
```

Waiting for #8399 to be merged to allow these to be used.

Reviewer: Actually alt isn't valid on `video` either, so should remove that too.
